relayer/chain/ethereum/listener.go

```diff
@@ -32,30 +28,23 @@ type Listener struct {
 	basicOutboundChannel        *outbound.BasicOutboundChannel
 	incentivizedOutboundChannel *outbound.IncentivizedOutboundChannel
 	mapping                     map[common.Address]string
-	db                          *store.Database
 	address                     common.Address
-	lightClientBridge           *lightclientbridge.Contract
 	messages                    chan<- []chain.Message
-	beefyMessages               chan<- store.BeefyRelayInfo
-	dbMessages                  chan<- store.DatabaseCmd
 	headers                     chan<- chain.Header
 	blockWaitPeriod             uint64
 	log                         *logrus.Entry
 }
 
-func NewListener(config *Config, conn *Connection, db *store.Database, messages chan<- []chain.Message,
-	beefyMessages chan<- store.BeefyRelayInfo, dbMessages chan<- store.DatabaseCmd, headers chan<- chain.Header,
+func NewListener(config *Config, conn *Connection, messages chan<- []chain.Message,
```

Comment:
small nit: for these long method signatures, lets break the signature along multiple lines

Reply:
@vgeddes can't gofmt do this for us?